aten/src/ATen/native/native_functions.yaml

```diff
@@ -104,6 +104,13 @@
 - func: _unpack_dual(Tensor(a) dual, int level) -> (Tensor(a) primal, Tensor tangent)
   variants: function
 
+# _new_zeros_with_same_feature_meta needs to be CompositeExplicitAutograd for
+# jvp support in functorch
+- func: _new_zeros_with_same_feature_meta(Tensor self, Tensor other, int? self_num_batch_dims=None) -> Tensor
```

Comment:
It would be nice to document this function a little more (e.g. write more about what it does)